lib/membrane_wav/serializer.ex

```diff
@@ -2,8 +2,7 @@ defmodule Membrane.WAV.Serializer do
   @moduledoc """
   Element responsible for raw audio serialization to WAV format.
 
-  Creates WAV header (its description can be found with `Membrane.WAV.Parser`) from received caps
-  and puts it before audio samples. The element assumes that audio is in PCM format.
+  Creates WAV header (its description can be found with `Membrane.WAV.Parser`) from received formats and puts it before audio samples. The element assumes that audio is in PCM format.
```

Comment:
IMHO the most accurate description, for the time being, would be:
```suggestion
  Creates WAV header (its description can be found with `Membrane.WAV.Parser`) based on a format received in caps
  and puts it before audio samples. The element assumes that audio is in PCM format.
```